src/screens/Game/GameSubMenu/index.tsx

```diff
@@ -244,6 +244,10 @@ export default function GamesSubmenu({
     setSteamRefresh(false)
   }
 
+  async function handleGogSetup() {
+    await ipcRenderer.invoke('runGOGSetupForGame', appName)
```

Comment:
We need a state here like we have for add to steam and add shortcuts. Was it already setup? Show a circle during setup...

Reply:
I added this button in this PR too, since we can't really tell if it was already ran in Bottles or Crossover